libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -65,7 +80,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
     // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:UseBest,2:Blend
+    // @RebootRequired: True
```

Comment:
If we fix the arming check problem I believe this can no longer be a reboot required flag?